docs/api-reference/next.config.js/headers.md

```diff
@@ -0,0 +1,91 @@
+---
+description: Add custom HTTP headers to your Next.js app.
+---
+
+# Headers
+
+Headers allow you to set custom HTTP headers for an incoming request path.
+
+To use Redirects you can use the `headers` key in `next.config.js`:
+
+```js
+module.exports = {
+  async headers() {
+    return [
+      {
+        source: '/about',
+        headers: [
+          {
+            key: 'x-custom-header',
+            value: 'my custom header value',
+          },
+          {
+            key: 'x-another-custom-header',
+            value: 'my other custom header value',
+          },
+        ],
+      },
+      ,
+    ]
+  },
+}
+```
+
+`headers` is an `async` function that expects an array to be returned holding objects with `source` and `headers`.
```

Comment:
Seems odd to codify `async` but not `function`. Let's just drop 'em?
Also add properties suffix.

```suggestion
`headers` is an async function that expects an array to be returned holding objects with `source` and `headers` properties:
```